android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -69,6 +69,11 @@ private Boolean isTablet() {
     return layout == Configuration.SCREENLAYOUT_SIZE_LARGE || layout == Configuration.SCREENLAYOUT_SIZE_XLARGE;
   }
 
+  @ReactMethod
+  public void getDeviceInfo(@NonNull final Promise promise) {
+    promise.resolve(this.getConstants().get("DeviceInfo"));
+  }
+
   @Override
   public @Nullable Map<String, Object> getConstants() {
```

Comment:
Should these be left as constants if they're liable to change?

Seems like this shouldn't be an overridden method if we expect to call these dynamically through the `getDeviceInfo` method call